### PR TITLE
more fixes

### LINKED ISSUE #146: erase gold in fid alignment protocol is not working
the positions of erased gold are wrong

### PR DESCRIPTION
**Pull request by azazellochg**
- Erase gold option now works for fid. alignment protocol (close #146 )
- fix wrong output file index numbers in the exclude views protocol, also do not copy objId
- fix rotation angle range for swapping x/y for newstack
- do not copy objId for apply TM protocol when excluding views
- fix output size in apply TM protocol
- add a warning for ctf corr. protocol if inputting ts with no alignment
- remove ts interpolation before estimating ctf

### CODE REVIEW

File: imod/protocols/protocol_applyTransformationMatrix.py [hunk 1]
```diff
@@ -110,9 +110,10 @@ def computeAlignmentStep(self, tsObjId):
         # Check if rotation angle is greater than 45º. If so,
         # swap x and y dimensions to adapt output image sizes to
         # the final sample disposition.
-        if rotationAngleAvg > 45 or rotationAngleAvg < -45:
+        if 45 < abs(rotationAngleAvg) < 135:
```
Comment (pconesa) [inline imod/protocols/protocol_applyTransformationMatrix.py:113]: Are we sure about this change (asking from my huge ignorance)?

**Comment by fede-pe** [comment]
test passing in my machine after [Merge pull request](https://github.com/scipion-em/scipion-em-imod/pull/157/commits/246392e4d1fe03f9e123a7f55a4c58ecc9259801) https://github.com/scipion-em/scipion-em-imod/pull/158 [from scipion-em/do-not-interpolate-ts](https://github.com/scipion-em/scipion-em-imod/pull/157/commits/246392e4d1fe03f9e123a7f55a4c58ecc9259801)

File: imod/protocols/protocol_applyTransformationMatrix.py (hunk 1 above)
Comment (azazellochg) [reply imod/protocols/protocol_applyTransformationMatrix.py:113]: yes, this is taken from copytomocoms

**Review by pconesa — APPROVED** [review]

**Comment by pconesa** [comment]
should we go ahead?

**Pull request merged**